streaming/src/main/scala/org/apache/spark/streaming/StreamingConf.scala

```diff
@@ -26,135 +26,162 @@ object StreamingConf {
 
   private[streaming] val BACKPRESSURE_ENABLED =
     ConfigBuilder("spark.streaming.backpressure.enabled")
+      .version("1.5.0")
```

Comment:
SPARK-9967 and SPARK-10099, commit ID: 392bd19d678567751cd3844d9d166a7491c5887e#diff-1b584c4ed88a9022abb11d594f760997